libs/application/templates/children-residence-change/src/forms/ContractRejected.ts

```diff
@@ -0,0 +1,49 @@
+import {
+  buildForm,
+  buildSection,
+  Form,
+  FormModes,
+  buildCustomField,
+} from '@island.is/application/core'
+import Logo from '../../assets/Logo'
+import * as m from '../lib/messages'
+
+export const ParentBContractRejected: Form = buildForm({
```

Comment:
Like discussed on a call, we can extract this object to a function and pass in the component ID